src/anchorfm-pupeteer/index.js

```diff
@@ -31,6 +36,7 @@ async function setPublishDate(page, date) {
     while (currentDate !== dateForComparison) {
       await clickSelector(page, navigationButtonSelector);
       currentDate = await getTextContentFromSelector(page, currentDateCaptionElementSelector);
+      await sleepSeconds(0.5);
```

Comment:
Why a sleep here?

Reply:
Well, I found out that for scheduling dates that are quite distant(in therms of months) it requires a lot of clicking in the calendar, and sometimes youtube-to-anchorfm would simply stuck while trying to set the publish date. That's why I included the sleep.

Reply:
I see but wouldn't be better to have a waitFor instead of a static wait? So that we can avoid any kind of lag issue

Reply:
I've tried adding a waitFor, something like this:

```javascript
    while (currentDate !== dateForComparison) {
      await clickSelector(page, navigationButtonSelector);
      await page.waitForSelector(currentDateCaptionElementSelector);
      currentDate = await getTextContentFromSelector(page, currentDateCaptionElementSelector);
    }
```

And youtube-to-anchorfm got stuck while trying to schedule this video

Reply:
If this solution is working, we can deep dive into something else in the future to be more async. I'm only afraid about a static wait could eventually breaks for any sort of lag or little change kn the behavior.

I'm ok with that since you've tested it and seems working nice. Approving